html/index.html

```diff
@@ -10,42 +10,42 @@
 </head>
 <body>
   <div class="navbar">
-    <h1 class="nav-wolox">Developer Tools</h1>
+    <h1 class="nav-wolox"><span>Developer Tools</span></h1>
```

Comment:
no need to add a span here

Reply:
It was the only way i could make the text disappear and not the image (because it is the background of .nav-wolox)